index.html

```diff
@@ -20,330 +20,334 @@
+                                    <li><button class="default-btn">Bootstrap</button></li>
+                                    <li><button class="default-btn">Ruby</button></li>
+                                </ul>
+                            </section>
 
-                        <div class="multiple-btn-container">
-                            <button class="smallprimarybtn">See Project</button>
+                            <div class="multiple-btn-container">
+                                <button class="smallprimarybtn">See Project</button>
+                            </div>
                         </div>
                     </div>
-                </div>
 
-            </div>
+                </div>
 
 
-        </setion>
+            </setion>
 
 
 
-        <section class="card-section">
-            <!-- PROFESSION ART PRINTING -->
-            <section class="professional_art_printing-section">
-                <div class="professional_art_printing-section-items">
-                    <div class="art_print_container">
-                        <h2 class="art_print_header">Profesional Art Printing Data</h2>
-                        <p class="art_print_paragraph">
-                            A daily selection of privately personalized reads; no accounts or
-                            sign-ups required. Has been the industry's standard.
-                        </p>
-                        <ul class="art_print_btn_container">
-                            <li><button class="transparent-btn">HTML</button></li>
-                            <li><button class="transparent-btn">Bootstrap</button></li>
-                            <li><button class="transparent-btn">Ruby</button></li>
-                        </ul>
-                    </div>
-                    <div class="art-main-btn-container">
-                        <button class="primarybtn">See Project</button>
+            <section class="card-section">
+                <!-- PROFESSION ART PRINTING -->
+                <section class="professional_art_printing-section">
+                    <div class="professional_art_printing-section-items">
+                        <div class="art_print_container">
+                            <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                            <p class="art_print_paragraph">
+                                A daily selection of privately personalized reads; no accounts or
+                                sign-ups required. Has been the industry's standard.
+                            </p>
+                            <ul class="art_print_btn_container">
+                                <li><button class="transparent-btn">HTML</button></li>
+                                <li><button class="transparent-btn">Bootstrap</button></li>
+                                <li><button class="transparent-btn">Ruby</button></li>
+                            </ul>
```

Comment:
- [ Optional]  Well done implementing all the requirements however your card elements are not displayed as shown in the caption, in order to fix that issue I recommend deleting the color #ffff from `.professional_art_printing-section-items` in your style.css file.

![review](https://user-images.githubusercontent.com/53347966/233064389-9e1e42b9-3e1b-4e75-8d9b-acac36c9a000.JPG)